rfcs/0031-devices-api.md

```diff
@@ -41,7 +41,7 @@ In this example, you can see the diversity of RTOS implementations for drivers a
 [guide-level-explanation]: #guide-level-explanation
 
 ## User App
-`tvm_device_<device>_t`s are implemented for each RTOS or platform required, these are included by the user who chooses as appropriate for their application. Notably, to avoid dynamic allocation, the user must provide the `tvm_device_<device>_t` struct and initialise it rather than it being created and setup for them in the API. This is augmented by named functions for each device, examples in the case of the "woofles" accelerator:
+TVM presumes that the RTOS, platform, or user application defines a struct type `tvm_device_<device>_t`, these are included by the user who chooses an implementation as appropriate for their application. Notably, to avoid dynamic allocation, the user must provide the `tvm_device_<device>_t` struct and initialise it rather than it being created and setup for them in the API. TVM then expects an implementation of the named functions for each device, examples in the case of the "woofles" accelerator:
```

Comment:
could you also explicitly highlight where `<device>` comes from from a user PoV?

Reply:
Is this not covered by the example just below this block?

Reply:
Nope, that was not the question, amended the text to clarify the origin.